pkg/utils/custom_matchers.go

```diff
@@ -11,46 +11,46 @@ import (
 )
 
 type ServiceMatcher struct {
-	service config.Service
+	service *config.Service
 }
 
-func EquivalentService(service config.Service) types.GomegaMatcher {
+func EquivalentService(service *config.Service) types.GomegaMatcher {
```

Comment:
never written one of these before. neat, but seems like a lot of work.

Reply:
yea it wasn't fun to write. but worth it cause it simplified testing

Reply:
I am not quite sure what the implication of changing this into a pointer will be. but if we do it all of them have to be changed and i believe what we pass into ginkgo might change.